### ISSUE TITLE
Feature request: Complete current specs in one document

### ISSUE THREAD

**Issue by lemmon-714** [IntersectMBO/cardano-ledger#3050]
It seems right now to follow the format base specs + diffs for each era; would be easier to follow if there would exist a document containing the complete currently valid specs. The diff-specs are still useful however, particularly once one is up to speed.

**Comment by JaredCorduan** [comment]
I agree completely, we've talked a lot about this internally. The plan is to use https://github.com/input-output-hk/formal-ledger-specifications to create the full specs.

In the meantime, someone could create some kind of way to parse the latex specs and create the combined specs. I'd be afraid of anything that was not mechanical, though, for fear of drift and maintenance. My team does not have the capacity for doing this band-aid solution anytime soon, though. If someone from the community would want to take this on I would be happy! :)
